spec/integration/docker_rspec_helper.rb

```diff
@@ -0,0 +1,61 @@
+class RspecDocker
```

Comment:
I would rather name this as ```DockerHelper```, as this class is independent of the specs and is in reality helping setup and manage docker.

On the other side, I tend to see helpers living in their own directory, but this is mostly an organizational taste of mine, what do you think?